app/src/main/java/com/sunlightlabs/android/congress/fragments/BillListFragment.java

```diff
@@ -306,27 +308,26 @@ public LoadBillsTask(BillListFragment context, int page) {
 		public List<Bill> doInBackground(Void... nothing) {
 			try {
 				
-				Map<String,String> params = new HashMap<String,String>();
-				
 				switch (context.type) {
 				case BILLS_ALL:
-					return BillService.recentlyIntroduced(page, PER_PAGE);
+					return BillService.recentlyIntroduced(page);
 				case BILLS_ACTIVE:
-					return BillService.recentlyActive(page, PER_PAGE);
+					return BillService.recentlyActive(page);
 				case BILLS_LAW:
-					return BillService.recentlyLaw(page, PER_PAGE);
+					return BillService.recentlyLaw(page);
 				case BILLS_SPONSOR:
-					return BillService.recentlySponsored(context.sponsor.bioguide_id, page, PER_PAGE);
+					return BillService.recentlySponsored(context.sponsor.bioguide_id, page);
 				case BILLS_CODE:
-					params.put("bill_type", context.bill_type);
-					params.put("number", String.valueOf(context.number));
-					return BillService.where(params, page, PER_PAGE);
+				    int congress = Bill.currentCongress();
+				    String bill_id = context.bill_type + String.valueOf(context.number) + "-" + String.valueOf(congress);
+					List<Bill> matches = new ArrayList<Bill>();
+                    Bill bill = BillService.find(bill_id);
+                    if (bill != null) matches.add(bill);
+                    return matches;
```

Comment:
Bill code lookup is now locked to the current Congress, instead of searching across Congresses.